test/SlowTests/Issues/RavenDB-17872.cs

```diff
@@ -250,13 +239,37 @@ public async Task CompareExchangeMetadata_CreateAndTryToRemove()
             }
         }
 
+        [Fact]
+        public async Task Can_Modify_Empty_Metadata()
+        {
+            using (var store = GetDocumentStore())
+            {
+                const string id = "users/1";
+                const string compareExchangeKey = $"{ClusterTransactionCommand.RvnAtomicPrefix}/{id}";
```

Comment:
use ClusterTransactionCommand.GetAtomicGuardKey instead? no need to mark items as internal

Reply:
done